spec/models/user_spec.rb

```diff
@@ -1,13 +1,11 @@
 require 'rails_helper'
-RSpec.describe User, :type => :model do
-  subject do
-    @user1 = FactoryBot.create(:user)
-  end
+describe User, :type => :model do
+  subject { @user1 = create(:user) }
 
   describe '#valid attributes' do
     it 'is valid with valid attributes' do
```

Comment:
The description should be `'is valid'` then you can use a context '`with valid params`'. So `with valid params... is valid`